siac/hostcmd.go

```diff
@@ -141,11 +141,8 @@ func hostcmd() {
 
 	// convert accepting bool
 	accept := yesNo(is.AcceptingContracts)
-	// convert price to SC/TB/mo
-	price, err := modules.StoragePriceToHuman(is.MinimumStoragePrice)
-	if err != nil {
-		price = ^uint64(0)
-	}
+	// convert price from bytes/block to TB/Month
+	price := currencyUnits(is.MinimumStoragePrice.Mul(types.NewCurrency64(4320e12)))
```

Comment:
Even better, can you use the function in the modules package that has thorough testing? It's there so that we don't get mistakes like these where people omit units or make the calculation out of order.